docs/networking/native-ipv6-networking.md

```diff
@@ -80,6 +79,10 @@ While default IPv6 addresses are configured automatically, you will need to stat
 
 ### Debian/Ubuntu
 
+{: .note}
```

Comment:
Since this is true for all distributions, I'm moving it out of the Debibuntu section.